components/brave_rewards/resources/ui/components/grant.tsx

```diff
@@ -156,6 +157,10 @@ class Grant extends React.Component<Props, State> {
       tokens = convertProbiToFixed(grant.probi)
     }
 
+    if (grant.type === 'ads') {
```

Comment:
nit: logic is opposite to other places in the codebase where we would have the following code which removes the need to calculate the date for ads:

```
let date = ''
if (grant.type !== 'ads') {
    date = new Date(grant.expiryTime).toLocaleDateString()
}
```

Reply:
done